qp_qiime2/__init__.py

```diff
@@ -155,6 +172,10 @@
             # this is an odd one, first encountered:
             # feature-classifier fit-classifier-naive-bayes
             if ename == element.NOVALUE:
+                # As of qiime2-2022.2 nothing is filtered here, so let's raise
+                # an error so we can catch and review if this happens in the
+                # future
+                raise ValueError(f"[REVIEW] {pname} {mid} due to {ename}")
                 add_method = False
```

Comment:
if you raise ValueError here, add_method will never be set to False. Is this desired?

Reply:
yes, if this happens we want everything to stop and have someone look - cause it should never happen.

Reply:
duh! Now, now I understand what you mean ... yes.